wordpress-mobile/WordPress-Android
# Disable Widgets for ARC devices

Fixes #7525 

To test:

1. Install on Chrome OS device
2. Verify that the widget install promo dialog doesn't show up after visiting stats three times.
3. Verify that the widget install promo DOES show up on an android device

I have verified that this check is the correct one, even though it's unofficial.
